automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -4,7 +4,6 @@ import java.util.UUID
 
 import akka.actor.ActorSystem
 import akka.testkit.TestKit
-import cats.effect.Blocker
```

Comment:
Blocker has been removed in Cats 3, so all imports and other places it is used have removed it as well

https://typelevel.org/cats-effect/docs/migration-guide#blocker